### PR TITLE
Disks smartfridge

### PR DESCRIPTION
**Pull request by TheLeopold**
## About The Pull Request
Disks smartstorage. Needs to be mapped in, but Valo tweaking map, I can't make him wait another week.
изображение

## Why It's Good For The Game
qol. Reduces disks clutter.

## Testing
Spawned, filled, emptied

## Changelog
:cl:
add: Added disks smartstorage
imageadd: added filled states for disk storage
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
Why do the disks need to be cold

**Comment by Humonitarian** [comment]
Why do the disks need to be cold

**Comment by TheLeopold** [comment]
Why do the disks need to be cold

**Comment by Loldaddy5676** [comment]
Why do the disks need to be cold

File: code/game/machinery/kitchen/smartfridge.dm [hunk 1]
```diff
@@ -35,6 +35,24 @@
...
+	desc = "When you need disks fast!"
+	icon_state = "smartfridge"
+	icon_fill10 = "diskfridge-fill10"
+	icon_fill20 = "diskfridge-fill20"
+	icon_fill30 = "diskfridge-fill30"
+
+/obj/machinery/smartfridge/disks/accept_check(var/obj/item/O as obj)
+	if(istype(O,/obj/item/computer_hardware/hard_drive/portable))
+		return 1
+	return 0
```
Comment (hyperioo) [inline code/game/machinery/kitchen/smartfridge.dm:52]:
```suggestion
		return TRUE
	return FALSE
```

**Review by hyperioo — APPROVED** [review]

**Review by Humonitarian — CHANGES_REQUESTED** [review]
Make it not require the same amount of power a fridge requires.

**Comment by Pink-Chink** [comment]
Fine for now. We should think about getting a non-fridge looking sprite in the future.

**Pull request merged**